lib/jormungandr/test/integration/Cardano/Wallet/Jormungandr/NetworkSpec.hs

```diff
@@ -301,46 +302,34 @@ spec = do
                     Left (ErrGetBlockNetworkUnreachable _) -> True
                     _ -> False
 
+
+    utxoWitness <- runIO $ generate arbitrary
```

Comment:
Instead of generating arbitrary witnesses for each `SignedTransactions`, we now only generate one, and use it for all inputs.

The witness contents themselves doesn't matter in these tests, so I think we should be fine.